examples/connext_dds/multichannel/c++11/market_data_publisher.cxx

```diff
@@ -9,170 +9,178 @@
  use the software.
  ******************************************************************************/
 
-#include <iostream>
+#include <dds/pub/ddspub.hpp>
+#include <rti/util/util.hpp>      // for sleep()
+#include <rti/config/Logger.hpp>  // for logging
 
+#include "application.hpp"  // for command line parsing and ctrl-c
 #include "market_data.hpp"
-#include <dds/dds.hpp>
 
-using namespace dds::core;
-using namespace rti::core;
-using namespace rti::core::policy;
-using namespace dds::domain;
-using namespace dds::topic;
-using namespace dds::pub;
-using namespace dds::pub::qos;
-
-MultiChannel create_multichannel_qos()
+rti::core::policy::MultiChannel create_multichannel_qos()
 {
     // We set the publish as multichannel using the different
     // channels to send different symbols. Every channel have a IP to send the
     // data.
 
     // Create 8 channels based on Symbol.
-    std::vector<ChannelSettings> channels;
-    channels.push_back(ChannelSettings(
-            std::vector<TransportMulticastSettings>(
+    std::vector<rti::core::ChannelSettings> channels;
+    channels.push_back(rti::core::ChannelSettings(
+            std::vector<rti::core::TransportMulticastSettings>(
```

Comment:
Replace with initializer_list:
```
channels.push_back(rti::core::ChannelSettings({ rti::core::TransportMulticastSettings( {}, "239.255.0.2", 0)));
```

This might work as well:

```
channels.push_back({
    { 
        { {}, "239.255.0.2", 0 }
    }
});
```